lib/libc/minimal/source/stdout/prf.c

```diff
@@ -759,8 +749,7 @@ int z_prf(int (*func)(), void *dest, char *format, va_list vargs)
 						}
 					} else {
 						/* Right justify */
-						(void) memmove((buf + (width - c)), buf, (size_t) (c
-										+ 1));
+						memmove(buf + (width - c), buf, c + 1);
```

Comment:
please keep `(void)` cast, this means that we are explicitly ignoring the return of this function. The `size_t` cast is also necessary since `c` is signed int.